### DISCUSSION TITLE
Move assertions on constants to only run at compile

### DISCUSSION THREAD

**Discussion by c-git** [rust-practice/cargo-leet#26] [Ideas]
Investigate using https://docs.rs/static_assertions/latest/static_assertions/

**Comment by c-git** [comment]
Maybe a test to confirm these would be enough and we don't need to actually have it checked on every compile

**Reply by sak96** [reply]
or we could have a testcase with the same assertion
that way while merge we have check it

**Comment by c-git** [comment]
Yeah I agree, I only left the debug_assert to be able to show where it is used

**Comment by c-git** [comment]
Implemented in test cases. See [PR](https://github.com/rust-practice/cargo-leet/pull/39)
